From 43a5ecb7858395fc5f2eab48d06d617672081df1 Mon Sep 17 00:00:00 2001
From: Johannes Demel <demel@uni-bremen.de>
Date: Sun, 15 Jan 2023 11:57:12 +0100
Subject: [PATCH 2/5] license: Fix SPDX identifiers

Some files still carried GPL identifiers. These were identified and
updated. Thus, all files should carry the LGPL identifier now.

Files with incorrect identifiers were found with:
`grep -irnI --exclude-dir={build,.git,cpu_features} GPL . | grep -v LGPL`

Fixes #613

Signed-off-by: Johannes Demel <demel@uni-bremen.de>
---
 CMakeLists.txt                           | 2 +-
 apps/CMakeLists.txt                      | 2 +-
 cmake/Modules/VolkConfig.cmake.in        | 2 +-
 cmake/Modules/VolkConfigVersion.cmake.in | 2 +-
 cmake/cmake_uninstall.cmake.in           | 2 +-
 docs/CMakeLists.txt                      | 2 +-
 include/volk/volk_version.h.in           | 2 +-
 lib/CMakeLists.txt                       | 2 +-
 lib/constants.c.in                       | 2 +-
 python/volk_modtool/CMakeLists.txt       | 2 +-
 10 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a6e4f87..2ee2c72 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@
 #
 # This file is part of VOLK
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 ########################################################################
diff --git a/apps/CMakeLists.txt b/apps/CMakeLists.txt
index 8ac9187..db6ab50 100644
--- a/apps/CMakeLists.txt
+++ b/apps/CMakeLists.txt
@@ -3,7 +3,7 @@
 #
 # This file is part of VOLK
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 ########################################################################
diff --git a/cmake/Modules/VolkConfig.cmake.in b/cmake/Modules/VolkConfig.cmake.in
index 3ffdf36..f18e07f 100644
--- a/cmake/Modules/VolkConfig.cmake.in
+++ b/cmake/Modules/VolkConfig.cmake.in
@@ -2,7 +2,7 @@
 #
 # This file is part of VOLK.
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 get_filename_component(VOLK_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
diff --git a/cmake/Modules/VolkConfigVersion.cmake.in b/cmake/Modules/VolkConfigVersion.cmake.in
index b6e9b51..b97d25d 100644
--- a/cmake/Modules/VolkConfigVersion.cmake.in
+++ b/cmake/Modules/VolkConfigVersion.cmake.in
@@ -2,7 +2,7 @@
 #
 # This file is part of VOLK.
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 set(MAJOR_VERSION @VERSION_INFO_MAJOR_VERSION@)
diff --git a/cmake/cmake_uninstall.cmake.in b/cmake/cmake_uninstall.cmake.in
index d9d13ea..7ffbc90 100644
--- a/cmake/cmake_uninstall.cmake.in
+++ b/cmake/cmake_uninstall.cmake.in
@@ -2,7 +2,7 @@
 #
 # This file is part of VOLK.
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 # http://www.vtk.org/Wiki/CMake_FAQ#Can_I_do_.22make_uninstall.22_with_CMake.3F
diff --git a/docs/CMakeLists.txt b/docs/CMakeLists.txt
index 90eb1d7..7c4c5e0 100644
--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -2,7 +2,7 @@
 #
 # Copyright 2022 Johannes Demel.
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 find_package(Doxygen)
diff --git a/include/volk/volk_version.h.in b/include/volk/volk_version.h.in
index 46013da..e565048 100644
--- a/include/volk/volk_version.h.in
+++ b/include/volk/volk_version.h.in
@@ -4,7 +4,7 @@
  *
  * This file is part of VOLK
  *
- * SPDX-License-Identifier: GPL-3.0-or-later
+ * SPDX-License-Identifier: LGPL-3.0-or-later
  */
 
 #ifndef INCLUDED_VOLK_VERSION_H
diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index 75055ee..3cfffe5 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -3,7 +3,7 @@
 #
 # This file is part of VOLK.
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 ########################################################################
diff --git a/lib/constants.c.in b/lib/constants.c.in
index fba8c39..049bc04 100644
--- a/lib/constants.c.in
+++ b/lib/constants.c.in
@@ -4,7 +4,7 @@
  *
  * This file is part of VOLK
  *
- * SPDX-License-Identifier: GPL-3.0-or-later
+ * SPDX-License-Identifier: LGPL-3.0-or-later
  */
 
 #if HAVE_CONFIG_H
diff --git a/python/volk_modtool/CMakeLists.txt b/python/volk_modtool/CMakeLists.txt
index aff7a62..dcc79a5 100644
--- a/python/volk_modtool/CMakeLists.txt
+++ b/python/volk_modtool/CMakeLists.txt
@@ -3,7 +3,7 @@
 #
 # This file is part of VOLK
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-License-Identifier: LGPL-3.0-or-later
 #
 
 ########################################################################
-- 
2.39.2

